### ISSUE TITLE
Resetting the database after restarting the server

### ISSUE THREAD

**Issue by alekcey0211** [skyrim-multiplayer/skymp#51]
When server configured using file or mongo databases, after each restarting the server, the database is reset and you have to start all over again

**Comment by alekcey0211** [comment]
update: I looked at what is written in the database during the game, and realized that nothing is written there since the start.

After switch to chakra i got this error after each start game
```
Error in server.tick: Error: 'JsGetGlobalObject' returned error 0x10003
Error: 'JsGetGlobalObject' returned error 0x10003
    at C:\Games\skymp-workspace\server\skymp5-server.js:87382:16
    at Generator.next (<anonymous>)
    at fulfilled (C:\Games\skymp-workspace\server\skymp5-server.js:87243:24)
```

Maybe after this error write to database stopped

**Comment by Pospelove** [comment]
Closing in favor of #53. Feel free to reopen or open a new issue if the problem is unrelated once we fix #53
